vcpkg/users/platforms/android.md

```diff
@@ -230,7 +230,7 @@ The folder [vcpkg_android_example_cmake_script](https://github.com/Microsoft/vcp
 
 [!INCLUDE [experimental](../../../includes/experimental.md)]
 
-vcpkg can export android archives ([AAR files](https://developer.android.com/studio/projects/android-library)). Once an archive is created, it can imported in Android Studio as a native dependent.  The archive is automatically consumed using [android studio's prefab tool](https://github.com/google/prefab).
+vcpkg can export Android archives ([AAR files](https://developer.android.com/studio/projects/android-library)). Once an archive is created, it can imported in Android Studio as a native dependent.  The archive is automatically consumed using [Android Studio's prefab tool](https://github.com/google/prefab).
```

Comment:
```suggestion
vcpkg can export Android archives ([AAR files](https://developer.android.com/studio/projects/android-library)). Once an archive is created, it can be imported in Android Studio as a native dependent.  The archive is automatically consumed using [Android Studio's prefab tool](https://github.com/google/prefab).
```